config/styleguide/context/redirectToDark.js

```diff
@@ -0,0 +1,11 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+/* eslint-disable */
```

Comment:
It didn't like something you were doing?

Reply:
copy pasta from the action.js file.
But just checked, and it doesn't like document and window. 
```
[eslint] 'document' is not defined. (no-undef)
```

Reply:
Ah, we probably need to tell `eslint` that we're in a browser environment:

```json
{
    "env": {
        "browser": true
    }
}
```